retrieval/company_info.py

```diff
@@ -26,11 +26,11 @@ def __init__(self, company_number: str, timestamp: str, authentication_fp: str =
         
         self._company_data = ChAPI.getChData(self._company_url, self.__api_key)
         # Links
-        links = self._company_data.get('links', dict())
-        self._officers_url = urljoin(self._base_url, links.get('officers', ''))
-        self._filing_history_url = urljoin(self._base_url, links.get('filing_history', '')) 
-        self._charges_url = urljoin(self._base_url, links.get('charges', '')) 
-        self._persons_significant_control_url = urljoin(self._base_url, links.get('persons_with_significant_control', ''))
+        self.links = self._company_data.get('links', dict())
```

Comment:
self.links should be self._links and the same should be the case in the following four lines